Core/Core/UITestHelpers/UITestHelpers.swift

```diff
@@ -147,6 +152,10 @@ public class UITestHelpers {
             ()
         case .mockNow(let date):
             Clock.mockNow(date)
+        case .enableExperimentalFeatures(let features):
+            for feature in features {
+                ExperimentalFeature(rawValue: feature)!.isEnabled = true
+            }
```

Comment:
There should be a way to reset this so it doesn't mess up neighboring tests. Either in reset() or disabling unset features here would probably work